### DISCUSSION TITLE
support to addd rbd trash purge schedule?

### DISCUSSION THREAD

**Discussion by zhucan** [ceph/ceph-csi#4387] [Q&A]
I want to delete the rbd volume from trash with specify time

**Comment by Madhu-1** [comment]
You can use rbd CLI commands to remove images from the trash (if they still exist) we don't support any settings in CSI which will allow users to keep the images in the trash for a specific time when the PVC/PV is deleted.

**Reply by zhucan** [reply]
is there a plan to support it?

**Reply by Madhu-1** [reply]
No we dont have any plan to support this one.
